src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -84,6 +84,10 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>NuGetResources.resx</DependentUpon>
     </Compile>
+    <!-- nuget.exe localized assemblies -->
+    <EmbeddedResource Include="$(OutputPath)\**\$(AssemblyName).resources.dll" Exclude="@(MergeExclude)" />
+    <!-- NuGet.Commands localized assemblies -->
+    <EmbeddedResource Include="$(OutputPath)\**\NuGet.Commands.resources.dll" Exclude="@(MergeExclude)" />
```

Comment:
Does this accidentally merge all frameworks of NuGet.Commands localized assemblies or no?

Reply:
No, it only merges net472 resource assemblies in NuGet.CommandLine case; because this OutputPath refers to NuGet.CommandLine output paths.

![NuGet.CommandLine OutputPath property](https://user-images.githubusercontent.com/1192347/190833747-5ca159cc-fa22-4461-a2aa-523883f8775e.png)

Reply:
Resovling.